src/Utilities.jl

```diff
@@ -5,6 +5,66 @@ module Utilities
 """
 Utilities
 
+export isexported, iscategory
+
+import Docile.Collector: QualifiedSymbol, Aside
+
+import Docile.Utilities: isexpr
+
+import Docile: Cache
+
 using Compat
 
+convert(::Type{UTF8String}, sym::Symbol)    = utf8(string(sym))
+convert(::Type{UTF8String}, s::ASCIIString) = utf8(s)
+
+function utf8checked!(d::Dict{Symbol, Any}, k)
+    isa(d[k], UTF8String) ? d[k] : (d[k] = convert(UTF8String, d[k]); d[k])
+end
+
+"""
+Get the ``Symbol`` representing an object such as ``Function`` or ``Method``.
+"""
+nameof(s::Symbol) = s
+nameof(m::Module, obj::Symbol)            = obj
+nameof(m::Module, obj::Module)            = module_name(obj)
+nameof(m::Module, obj::Method)            = nameof(Cache.getmeta(m, obj)[:code])
+nameof(m::Module, obj::DataType)          = obj.name.name
+nameof(m::Module, obj::QualifiedSymbol)   = symbol(last(split(string(obj), '.')))
+
+function  nameof(x::Expr)
+    isa(x.args[1], Bool)  ?
+        nameof(x.args[2]) :
+        isexpr(x.args[1], :(.)) ?
+                      x.args[1] :
+                      nameof(x.args[1])
```

Comment:
Yes looks nicer ;) changed

I took that from [your other code](https://github.com/MichaelHatherly/Docile.jl/blob/master/src/Legacy/atdoc.jl#L40-46)

Maybe this should also be changed or just leave it as it is the legacy code and might be removed in the future?